README.md

```diff
@@ -35,6 +35,9 @@ To determine whether to allow response _response_ to a request _request_, run th
    1. Append (_request_'s opaque media identifier, _request_'s current URL) to the user agent's opaque-safelisted requesters set.
    1. Return true.
 1. If _nosniff_ is true, then return false.
+1. If _response_'s status is not an [ok status](https://fetch.spec.whatwg.org/#ok-status), then return false.
```

Comment:
I wonder if we could also special-case 204 and 206 here?

Reply:
Special case in what way?

Reply:
206 => return `false` (i.e. not allowed).  Rationale: 206 is for partial content responses, which should 1) never be needed for scripts/javascript and 2) should only be needed for audio/video which is covered by "opaque-safelisted requesters set".

204 => not sure about it anymore, but I thought that if we can guarantee that 204 has an empty response body, then we can go ahead and allow the response?  or maybe this should be more general about allowing a response with an empty body?

Reply:
Ah yeah, I have some logic from 206 in step 6 based on CORB, but if we want to make that more general let's discuss it in an issue? Same for 204/empty body I suppose.